prime-router/metadata/hl7_mapping/common/message-header.yml

```diff
@@ -110,6 +110,11 @@ elements:
     value: [ '%resource.code' ]
     hl7Spec: [ MSH-11-2 ]
 
+  - name: version-id
```

Comment:
The only thing that I feel confident in saying is that we'll set MSH-12 to `2.5.1` by default and the receiver can interpret that how they would like.   I specifically added this because now that the HL7 message class used is the 27 one, that is what will be put in MSH-12 without this change.  Since, the goal for the ELIMS message was  `in == out` I made this update.

However, I do think this should lead to a larger conversation around what is meant by these different versions and what that means for report stream